src/main/java/de/hysky/skyblocker/skyblock/crimson/slayer/FirePillarAnnouncer.java

```diff
@@ -0,0 +1,38 @@
+package de.hysky.skyblocker.skyblock.crimson.slayer;
+
+import de.hysky.skyblocker.utils.SlayerUtils;
+import de.hysky.skyblocker.utils.Utils;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.decoration.ArmorStandEntity;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class FirePillarAnnouncer {
+
+    private static final Pattern FIRE_PILLAR_PATTERN = Pattern.compile("(\\d+)s \\d+ hits");
+
+    public static void CheckFirePillar(Entity entity) {
```

Comment:
Please follow `camelCase` convention for method names. Same for the method below.